### PR TITLE
remarkable: add more detail to README about launcher

### PR DESCRIPTION
**Pull request by tcrs**
This adds a little more detail about how to use koreader with a different launcher on reMarkable

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6397)

### CODE REVIEW

File: platform/remarkable/README_remarkable.txt [hunk 1]
```diff
@@ -13,12 +13,35 @@ with a USB cable.
    ssh root@<your-remarkable>
 
    unzip <koreader-package>.zip
+
+# Launching using the included launcher
+
+koreader includes a very simple launcher called `button-listen` which starts
```
Comment (Frenzie) [inline platform/remarkable/README_remarkable.txt:19]:
```suggestion
KOReader includes a very simple launcher called `button-listen`, which starts
```

File: platform/remarkable/README_remarkable.txt [hunk 2]
```diff
@@ -13,12 +13,35 @@ with a USB cable.
...
+koreader when the middle button is held down for 3 or more seconds in the
+xochitl UI.
```
Comment (Frenzie) [inline platform/remarkable/README_remarkable.txt:21]: Do people know what that is? A subtle hint like "the default xochitl UI" might be preferable.

File: platform/remarkable/README_remarkable.txt [hunk 3]
```diff
@@ -13,12 +13,35 @@ with a USB cable.
...
+
    cp -v koreader/*.service /etc/systemd/system/
    systemctl enable --now button-listen
 
-Hold down the middle button for 3 seconds to start koreader. To return to
-xochitl just exit koreader (swipe down from the top of the screen, select icon
-in the top right, Exit, Exit).
+To return to xochitl just exit koreader (swipe down from the top of the screen,
```
Comment (Frenzie) [inline platform/remarkable/README_remarkable.txt:26]:
```suggestion
To return to xochitl just exit KOReader (swipe down from the top of the screen,
```

File: platform/remarkable/README_remarkable.txt [hunk 4]
```diff
@@ -13,12 +13,35 @@ with a USB cable.
...
+Alternatively, if you want the default behaviour of koreader exit starting up
+xochitl you can run (once on installation):
+
+   cp -v koreader/koreader.service /etc/systemd/system/
+
+And then the launcher needs to run:
+
+   systemctl start koreader
+
+To start koreader (this is the command which `button-listen` runs).
```
Comment (Frenzie) [inline platform/remarkable/README_remarkable.txt:47]:
```suggestion
To start KOReader (this is the command which `button-listen` runs).
```

But it's probably clearer to just merge it into the sentence above.

File: platform/remarkable/README_remarkable.txt (hunk 1 above)
Comment (tcrs) [reply platform/remarkable/README_remarkable.txt]: Ah yes, I've corrected the captialisation of koreader->KOReader throughout

File: platform/remarkable/README_remarkable.txt (hunk 2 above)
Comment (tcrs) [reply platform/remarkable/README_remarkable.txt]: Good point, I've mentioned xochitl the first time and just used Default UI from then on.

**Pull request merged**